package.json

```diff
@@ -4,16 +4,16 @@
   "private": true,
   "description": "The deepest card game under the sea",
   "scripts": {
+    "_lint": "vue-cli-service lint --ext .js,.ts,.vue .",
     "build": "vue-cli-service build && cp ./client/public/favicon.ico ./assets",
     "test:unit": "vue-cli-service test:unit",
-    "lint": "vue-cli-service lint --no-fix",
+    "lint": "npm run _lint -- --no-fix",
+    "lint:fix": "npm run _lint -- --fix",
```

Comment:
Calls the vue cli lint script but passes fix so we actively need to choose to do that.